apps/bluetooth/transfer.html

```diff
@@ -38,11 +38,9 @@
       <section>
         <h1 data-l10n-id="confirmation">Confirmation</h1>
         <p>
-          <img id="icon" src="style/images/settings_bluetooth.png" alt="Bluetooth Icon"/>
-          <strong data-l10n-id="bluetooth">Bluetooth</strong>
           <small data-l10n-id="bluetooth-status-description">Bluetooth is disabled</small>
         </p>
-        <p data-l10n-id="turn-bluetooth-on">Do you want to turn Bluetooth on?</p>
+        <small data-l10n-id="turn-bluetooth-on">Do you want to turn Bluetooth on?</small>
```

Comment:
This probably shall be consistent with new text used in l10n file, so please update default text together with new l10nid key